paulfiesel/TD2-Stats
# test: add integration test for /healthz endpoint

- Moved app.py to `backend` folder
- Added a new test file `test_healthz.py` in the `tests` directory
- Implemented a pytest fixture `client` to create a test client for the Flask app
- Created a test function `test_healthz` to:
  - Send a GET request to the /healthz endpoint
  - Verify that the response status code is 200
  - Ensure that the response body is empty.
- This test ensures that the /healthz endpoint behaves correctly, providing a quick way to validate the application's health check functionality
